include/CL/opencl.hpp

```diff
@@ -4530,6 +4539,24 @@ class BufferRenderGL : public Buffer
     }
 };
 
+#if defined(cl_ext_image_requirements_info)
+enum class ImageRequirementsInfoExt : cl_image_requirements_info_ext
+{
+    ExtNone = 0,
```

Comment:
The new version is much better - thanks! - but there's still a small amount of redundancy since the full name will be:

`ImageRequirementsInfoExt::ExtNone`

Options:

1. Do nothing, leave it as it is.
2. Take `Ext` off of the enum class name: `ImageRequirementsInfo::ExtNone`.
3. Take `Ext` off of the enum name: `ImageRequirementsInfoExt::None`.

I have a slight preference for (3).  What do you think?

Reply:
Done thanks !